mne/event.py

```diff
@@ -885,3 +886,349 @@ def concatenate_events(events, first_samps, last_samps):
         events_out = np.concatenate((events_out, e2), axis=0)
 
     return events_out
+
+
+class ElektaAverager(object):
+    """Handles events and averaging categories defined in DACQ
+    (data acquisition) software of Elekta TRIUX/Vectorview systems.
+
+    Parameters
+    ----------
+    info : Info
+        An instance of Info where the DACQ parameters will be taken from.
+
+    Attributes
+    ----------
+    categories : list
+        List of averaging categories marked active in DACQ.
+    events : list
+        List of events that are in use (referenced by some averaging category).
+    reject : dict
+        Rejection criteria from DACQ that can be used with mne.Epochs.
+        Note that mne does not support all DACQ rejection criteria
+        (e.g. spike, slope)
+    flat : dict
+        Flatness criteria from DACQ that can be used with mne.Epochs.
+    """
+
+    # averager related DACQ variable names (without preceding 'ERF')
+    vars = ['magMax', 'magMin', 'magNoise', 'magSlope', 'magSpike', 'megMax',
+            'megMin', 'megNoise', 'megSlope', 'megSpike', 'eegMax', 'eegMin',
+            'eegNoise', 'eegSlope', 'eegSpike', 'eogMax', 'ecgMax', 'ncateg',
+            'nevent', 'stimSource', 'triggerMap', 'update', 'version',
+            'artefIgnore', 'averUpdate']
+
+    # event-related variables
+    event_vars = ['Name', 'Channel', 'NewBits', 'OldBits', 'NewMask',
+                  'OldMask', 'Delay', 'Comment']
+
+    # category-related variables
+    cat_vars = ['Comment', 'Display', 'Start', 'State', 'End', 'Event',
+                'Nave', 'ReqEvent', 'ReqWhen', 'ReqWithin',  'SubAve']
+
+    def __init__(self, info):
+        acq_pars = info['acq_pars']
+        if not acq_pars:
+            raise ValueError('No acquisition parameters')
+        self.acq_dict = _acqpars_dict(acq_pars)
+        if 'ERFversion' not in self.acq_dict:
+            raise ValueError('Cannot parse version. The file may be from'
+                             'DACQ <3.4 which is not supported yet')
+        # set instance variables
+        for var in ElektaAverager.vars:
+            val = self.acq_dict['ERF' + var]
+            if var[:3] in ['mag', 'meg', 'eeg', 'eog', 'ecg']:
```

Comment:
there is a private function in mne.channels IIRC, to get the data channel types (this regularly changes)